### PR TITLE
my last edits

### PR DESCRIPTION
**Pull request by Asel-K**
I've decided to go all over again through the assignment instructions and that's why had to give a new name instead of lesson-1-1.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    title: 'Submit the assignment',
+  },
+]
+ 
+function App() {
+ 
+  return (
+    <div>
+      <h1>Todo List</h1>
+
```
Comment (kaytijackson97) [inline src/App.js:23]: the blank lines on line 23, 28, 30, and 34 aren't needed.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <ul>
+      {todoList.map(function (item) {
+          return (
+            <li key={item.id}>
+              
+              <span>{item.title}</span>
```
Comment (kaytijackson97) [inline src/App.js:29]: you can take {item.title} out of the span tags and place them between the list tags on one line.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+]
+ 
+function App() {
+ 
+  return (
+    <div>
+      <h1>Todo List</h1>
+
+      <ul>
+      {todoList.map(function (item) {
```
Comment (kaytijackson97) [inline src/App.js:25]: the entire code block inside of the unordered list tags should be indented on place.

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  {
+    id: '2',
+    title: 'Complete the assignment',
+  },
+  {
+    id: '3',
+    title: 'Submit the assignment',
+  },
+]
```
Comment (kaytijackson97) [inline src/App.js:16]: should have a semi-colon at the end of the square bracket

**Review by kaytijackson97 — CHANGES_REQUESTED** [review]
great start! showing a good understanding of what React has to offer. Just a couple quick notes to clean up the code a bit to make it more readable and condensed. Your use of the span tag is a little unnecessary since you can wrap {item.title} in the list tag. My other comments are just good coding habits to get used to.

File: src/App.js (hunk 4 above)
Comment (Asel-K) [reply src/App.js:16]: Thank you for all your recommendations! very helpful. Ill try to follow them all.

**Pull request merged**